src/renderer/account/components/AccountPanel/AccountPanel.js

```diff
@@ -30,6 +29,18 @@ export default class AccountPanel extends React.Component {
     return `Address ${this.props.address}`;
   }
 
+  renderBalances = () => {
+    const balances = values(this.props.balances);
+
+    if (balances.length === 0) {
+      return 'No available assets.';
```

Comment:
I also added this when there are no assets.  That way we don't have to explicitly output NEO and GAS before tokens.